README.md

```diff
@@ -81,12 +81,22 @@ The following projects are official OpenJS Foundation projects. If you are inter
 |   | [vis.gl](https://vis.gl/) |   | [Contributing Guide](https://github.com/visgl/deck.gl/blob/HEAD/CONTRIBUTING.md) | [Code of Conduct](https://github.com/visgl/deck.gl/blob/HEAD/CODE_OF_CONDUCT.md) |
 
 
-### OpenJS Collaboration spaces
+### OpenJS Collaboration Spaces
 
 The following are official OpenJS Collaboration spaces.
 If you are interested in fostering collaboration with the support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression](./collaboration-spaces/COLLABORATION_SPACE_PROGRESSION.md)
 
-* There are currently no collaboration spaces.
+| Name                                         |  Repository                        | Description                                                     |
+| -------------------------------------------- |:----------------------------------:|-----------------------------------------------------------------|
+| Open Visualization (OpenVis)                 | [Repository][openvis]              | Visualization libraries based on JavaScript and WebGL           |
```

Comment:
Should Standards be in there as well?

Reply:
Nope it’s still a WG for now and listed below.